### ISSUE TITLE
can't signup through cmd line

### ISSUE THREAD

**Issue by acornejo** [exponent/exp#26]
```
# exp --version
0.30.0
```

```
# exp signup
[exp] Cannot read property 'apply' of undefined
TypeError: Cannot read property 'apply' of undefined
    at Command._callee$ (/opt/node-v6.3.1-linux-x64/lib/node_modules/exp/build/exp.js:276:29)
    at tryCatch (/opt/node-v6.3.1-linux-x64/lib/node_modules/exp/node_modules/regenerator-runtime/runtime.js:64:40)
    at GeneratorFunctionPrototype.invoke [as _invoke] (/opt/node-v6.3.1-linux-x64/lib/node_modules/exp/node_modules/regenerator-runtime/runtime.js:355:22)
    at GeneratorFunctionPrototype.prototype.(anonymous function) [as next] (/opt/node-v6.3.1-linux-x64/lib/node_modules/exp/node_modules/regenerator-runtime/runtime.js:116:21)
    at step (/opt/node-v6.3.1-linux-x64/lib/node_modules/exp/node_modules/babel-runtime/helpers/asyncToGenerator.js:17:30)
    at /opt/node-v6.3.1-linux-x64/lib/node_modules/exp/node_modules/babel-runtime/helpers/asyncToGenerator.js:35:14
    at new Promise (/opt/node-v6.3.1-linux-x64/lib/node_modules/exp/node_modules/core-js/library/modules/es6.promise.js:191:7)
    at Command.<anonymous> (/opt/node-v6.3.1-linux-x64/lib/node_modules/exp/node_modules/babel-runtime/helpers/asyncToGenerator.js:14:12)
    at Command.<anonymous> (/Users/Jesse/Exponent/newuniverse/universe/dev/exp/src/exp.js:47:10)
    at Command.listener (/opt/node-v6.3.1-linux-x64/lib/node_modules/exp/node_modules/commander/index.js:301:8)
    at emitTwo (events.js:111:20)
    at Command.emit (events.js:191:7)
    at Command.parseArgs (/opt/node-v6.3.1-linux-x64/lib/node_modules/exp/node_modules/commander/index.js:615:12)
    at Command.parse (/opt/node-v6.3.1-linux-x64/lib/node_modules/exp/node_modules/commander/index.js:458:21)
    at _callee3$ (/opt/node-v6.3.1-linux-x64/lib/node_modules/exp/build/exp.js:72:55)
    at tryCatch (/opt/node-v6.3.1-linux-x64/lib/node_modules/exp/node_modules/regenerator-runtime/runtime.js:64:40)
```

In particular I found the following path a bit surprising:

/Users/Jesse/Exponent/newuniverse/universe/dev/exp/src/exp.js:47:10

This is not a mac (/Users doesn't exist), and there is no user named Jesse. May be an unrelated leftover from the machine where exponent was built.

**Comment by dikaiosune** [comment]
@acornejo hi! Thanks for the report! It looks like you're running with Node 6.3.1, where we require at least the current LTS release for our tools. I believe the current LTS version is 6.9.1. Can you try upgrading and seeing if that resolves your issue?

Regarding the stray path from a Mac, this is an artifact of the current build process and how it handles sourcemaps. I believe we have a patch in the pipeline to get these paths correctly resolving to the local directory, but I'd need to double check to be sure when it's going to land & release.

**Comment by acornejo** [comment]
I tried with latest LTS (6.9.2), same results:

```
# exp signup
[exp] Cannot read property 'apply' of undefined
TypeError: Cannot read property 'apply' of undefined
    at Command._callee$ (/opt/node-v6.9.2-linux-x64/lib/node_modules/exp/build/exp.js:276:29)
    at tryCatch (/opt/node-v6.9.2-linux-x64/lib/node_modules/exp/node_modules/regenerator-runtime/runtime.js:64:40)
    at GeneratorFunctionPrototype.invoke [as _invoke] (/opt/node-v6.9.2-linux-x64/lib/node_modules/exp/node_modules/regenerator-runtime/runtime.js:355:22)
    at GeneratorFunctionPrototype.prototype.(anonymous function) [as next] (/opt/node-v6.9.2-linux-x64/lib/node_modules/exp/node_modules/regenerator-runtime/runtime.js:116:21)                                                                                                                                                               
    at step (/opt/node-v6.9.2-linux-x64/lib/node_modules/exp/node_modules/babel-runtime/helpers/asyncToGenerator.js:17:30)
    at /opt/node-v6.9.2-linux-x64/lib/node_modules/exp/node_modules/babel-runtime/helpers/asyncToGenerator.js:35:14
    at Promise.F (/opt/node-v6.9.2-linux-x64/lib/node_modules/exp/node_modules/core-js/library/modules/_export.js:35:28)
    at Command.<anonymous> (/opt/node-v6.9.2-linux-x64/lib/node_modules/exp/node_modules/babel-runtime/helpers/asyncToGenerator.js:14:12)
    at Command.<anonymous> (/Users/Jesse/Exponent/newuniverse/universe/dev/exp/src/exp.js:47:10)
    at Command.listener (/opt/node-v6.9.2-linux-x64/lib/node_modules/exp/node_modules/commander/index.js:301:8)
    at emitTwo (events.js:111:20)
    at Command.emit (events.js:191:7)
    at Command.parseArgs (/opt/node-v6.9.2-linux-x64/lib/node_modules/exp/node_modules/commander/index.js:615:12)
    at Command.parse (/opt/node-v6.9.2-linux-x64/lib/node_modules/exp/node_modules/commander/index.js:458:21)
    at _callee3$ (/opt/node-v6.9.2-linux-x64/lib/node_modules/exp/build/exp.js:72:55)
    at tryCatch (/opt/node-v6.9.2-linux-x64/lib/node_modules/exp/node_modules/regenerator-runtime/runtime.js:64:40)
```

**Comment by dikaiosune** [comment]
Quite right! I'll work on a fix for this. In the meantime, the node 7.2 series works for me to signup in exp, and you can still use XDE to create an account.

**Comment by dikaiosune** [comment]
Hi! Sorry about the delay here. I've published v0.30.1 to npm which should resolve this issue. Could you try installing that and confirming it fixed things for you?

**Comment by acornejo** [comment]
worked. thank you @dikaiosune!

closing.
